cli/pkg/k8s/k8s.go

```diff
@@ -29,7 +29,7 @@ import (
 
 var HOME = os.Getenv("HOME")
 
-func kubeInit() (kubernetes.Interface, *rest.Config) {
+func KubeInit() (kubernetes.Interface, *rest.Config) {
```

Comment:
Ignore this `KubeClient()` here in the upcoming PR `kubeclient` will be created in the tobs cmd level & will be passed to the `k8s` package.